apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1160,71 +1169,78 @@ export class WorkshopForm extends React.Component {
             </Col>
           </Row>
           <Row>
-            {this.state.course === 'Build Your Own Workshop' && (
-              <div style={styles.container}>
-                <div style={styles.extraMargin}>
-                  <label>Select Workshop Topic(s)</label>
-                  <div
-                    className="dropdown show"
-                    id={'topics'}
-                    onKeyDown={this.onKeyDown}
-                  >
-                    <button
-                      style={{...styles.fullWidth, ...styles.topicsButton}}
-                      className="btn btn-secondary dropdown-toggle"
-                      id="dropdownMenuButton"
-                      type="button"
-                      data-toggle="dropdown"
-                      aria-haspopup={true}
-                      aria-label="topics dropdown"
+            <FormGroup validationState={validation.style.course_offerings}>
+              {this.state.course === BuildYourOwnWorkshop && (
```

Comment:
Should this line be outside the FormGroup?

Reply:
Moved this and the <row!